### PR TITLE
FS-1981, FS-1982: UI bugs & translations

### PR DESCRIPTION
**Pull request by tferns**
Summary:

**FS-1981:** During 14caa96698425c33d04e2209990f7a543c662aeb it looks like a duplicate table was added while resolving merge conflicts, this has been fixed

**FS-1982:** 

- On the [index page](https://github.com/communitiesuk/funding-service-design-frontend/blob/441d2ce61ca597fdda2043925a37225f1fc82a77/app/templates/index.html#L39), this translation was wrapped in appropriate tags, but didn't have the translation in the [relevant .mo file](https://github.com/communitiesuk/funding-service-design-frontend/blame/bad05fd88192b9827a4c468728e1bd68e3ba089f/app/translations/cy/LC_MESSAGES/messages.po#L145)
- Noticed some dates weren't being translated on both pages, so have done that while I was here
- Contact details macro wasn't translated, so I did all that I could find on the spreadsheet

### CODE REVIEW

**Review by rayhan15243 — APPROVED** [review]
@tferns The translation for "Do not send any applications or attachments by email. We'll only accept applications through this service." looks correct to me. I'll approve the PR on the basis of this. 

@ramsharma-prog or @gio-karunakaran to check the bugfix for the rest e.g. applicant dashboard.

File: app/translations/cy/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -327,3 +314,52 @@ msgstr "Cyfeirnod"
...
+msgstr ""
+"Ni fyddwch yn gallu newid iaith unwaith y byddwch chi wedi dechrau eich "
+"cais."
+
+#: app/templates/partials/contact_details.html:3
+msgid "Email:"
+msgstr "Ebost:"
+
+#: app/templates/partials/contact_details.html:7
+#, fuzzy
```
Comment (gio-karunakaran) [inline app/translations/cy/LC_MESSAGES/messages.po:338]: I think we should get rid of fuzzy translations - they don't show up correctly when running the app - just delete the `fuzzy` line

**Review by gio-karunakaran — CHANGES_REQUESTED** [review]
great stuff - just a few changes :)

File: app/templates/dashboard.html [hunk 2]
```diff
@@ -76,30 +76,6 @@ <h1 class="govuk-heading-xl">{% trans %}Your applications{% endtrans %}</h1>
...
-    'text': 'Application Reference'
-    },
-    {
-    'text': 'Action'
-    }
-    ],
-    'rows': ns.rows
-    }) }}
-{% endif %}
-
```
[the remark this answers is not in the document]
Comment (tferns) [reply app/templates/dashboard.html:101]: The headers are duplicates of the translated versions, which are just above in the same file, looks like something went wrong during a merge conflict in https://github.com/communitiesuk/funding-service-design-frontend/commit/14caa96698425c33d04e2209990f7a543c662aeb causing them to be [duplicated](https://github.com/communitiesuk/funding-service-design-frontend/commit/14caa96698425c33d04e2209990f7a543c662aeb#diff-39b35bec7111a286dc3e119687cd779f3f2dfde61d886b5fa00d3ae4421424f6R54-R100), you can view a screenshot of this in Rayhan's bug here: https://digital.dclg.gov.uk/jira/browse/FS-1581 👍

[the remark this answers is not in the document]
Comment (gio-karunakaran) [reply app/templates/dashboard.html:101]: ahh perfect 🙌🏾 thank you!

**Review by gio-karunakaran — APPROVED** [review]
LGTM 👍🏾

**Pull request merged**